values-schema.yaml

```diff
@@ -3328,6 +3328,13 @@ properties:
               type: object
               additionalProperties: true
             default: []
+          monitoring:
+            type: object
+            properties:
+              disabledRules:
```

Comment:
I would like to separate monitoring from the app itself. 

In prometheus settings it is already possible to disable selected rules.
See: `apps.prometheus.disabledRules`